Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoISE.yaml

```diff
@@ -0,0 +1,128 @@
+    let EventOriginalTypeList = toscalar(EventFieldsLookup 
+        | summarize make_set(EventOriginalType));
+    let GetSrcIpAddr = (src_ip:string) {
+        case ( 
+            src_ip matches regex @"\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}", src_ip, ""
+        )
+    };
+    let GetMacAddr = (mac:string) {
+        case ( 
+            mac matches regex @"[a-fA-F0-9\-:]{17}", mac, ""
+        )
+    };
+    let CiscoISENSParser = (
+        starttime:datetime=datetime(null), 
+        endtime:datetime=datetime(null),
+        srcipaddr_has_any_prefix:dynamic=dynamic([]), 
+        dstipaddr_has_any_prefix:dynamic=dynamic([]), 
+        ipaddr_has_any_prefix:dynamic=dynamic([]),
+        dstportnumber:int=int(null), 
+        hostname_has_any:dynamic=dynamic([]), 
+        dvcaction:dynamic=dynamic([]), 
+        eventresult:string='*', 
+        disabled:bool=false){
+        let src_or_any=set_union(srcipaddr_has_any_prefix, ipaddr_has_any_prefix); 
+        let dst_or_any=set_union(dstipaddr_has_any_prefix, ipaddr_has_any_prefix);
+        let ip_any = set_union(srcipaddr_has_any_prefix, dstipaddr_has_any_prefix, ipaddr_has_any_prefix); 
+        Syslog
+        | where not(disabled)
+        | where (isnull(starttime) or TimeGenerated>=starttime) 
+            and (isnull(endtime) or TimeGenerated<=endtime) 
+        | where ProcessName has_any ("CISE", "CSCO")
+        | parse kind = regex SyslogMessage with @"\d{10}\s" EventOriginalType @"\s(NOTICE|INFO|WARN|WARNING|ERROR|FATAL|DEBUG)"
+        | where  EventOriginalType in (EventOriginalTypeList)
+            and (array_length(dvcaction) == 0)
```

Comment:
Now we have DvcAction so this field is also valid. Please move this to EventORiginalTypeList as well.